roles/openshift_metrics/tasks/install_metrics.yaml

```diff
@@ -11,19 +12,23 @@
   loop_control:
     loop_var: include_file
   when: "not {{ openshift_metrics_heapster_standalone | bool }}"
+  run_once: True
 
 - name: Install Heapster Standalone
   include: install_heapster.yaml
   when: "{{ openshift_metrics_heapster_standalone | bool }}"
+  run_once: True
```

Comment:
@sdodson I would agree we only need to generate certificates and run 'installing heapster' once.  I think removing it from the calling task is correct.